modules/coreboot

```diff
@@ -93,8 +93,7 @@ $(eval $(call coreboot_module,purism,24.02.01))
 
 # MSI and Nitropad NV41 / NS50 boards are based on Dasharo coreboot port
```

Comment:
Also please note ~upstream coreboot base from which the fork is derived from (so that we can reuse coreboot buuildstack if possible to reduce CI build time).

Reply:
Right, comment updated.